master/buildbot/status/web/mybuilds.py

```diff
@@ -16,7 +17,21 @@ def content(self, req, cxt):
             master.config.myBuildDaysCount,
         )
 
-        cxt['builds'] = builds
+        builds_by_ssid = {}
```

Comment:
I am really not sure, that it is a correct place for this operation. As we can read from djangobook (https://djangobook.com/model-view-controller-design-pattern/), the controller should decide who can have access to the model and defer it to a template. This code, in my personal opinion, should be put in models.

Reply:
In my opinion, the controller is a better place, because we can use DB functions again regardless final html code. In classical MVC (like Ruby on Rails) we can use more logic on view, but in Jinja this is ugly (we have to write dedicated tag / filter).

Maybe I should move this for-loop to another method and then I can test it.
@campos-ddc What do you think?

Reply:
I don't really care, perhaps try to follow the same pattern that the rest of `katana` code uses (if there is any)